src/questions/templates/questions/snippets/comment_answers.html

```
@@ -0,0 +1,53 @@
+          {{ answer.question.instructions|unwrap_p }}
+          {% if request.user.is_staff %}
+            {% if answer.question.solution_text %}
+              <p class="text-muted"><small><b>Solution:</b> {{ answer.question.solution_text|linebreaksbr }}</small></p>
+            {% endif %}
+            {% if answer.question.solution_file %}
+              <p class="text-muted"><small><b>Solution file:</b>
+                <a href="{{ answer.question.solution_file.url }}" target="_blank">{{ answer.question.solution_file|filename|safe }}</a></small></p>
+            {% endif %}
+            {% if answer.question.marker_notes %}
+              {# unwrap_p strips summernote's wrapping <p> so a one-line note sits beside its label, like Solution above. #}
+              {# A <div> (not <p>) wraps it because multi-paragraph notes keep their <p> tags, invalid nested in a <p>. #}
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </p></p></p></div></p></p> ], start tag match failed [ <p> ] on line 33.

(tag-pair)